src/containers/settings/animas/Animas.js

```diff
@@ -0,0 +1,171 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes } from 'react';
+
+import Header from '../header/Header';
+import Table from '../../../components/common/Table';
+import CollapsibleContainer from '../../common/CollapsibleContainer';
+
+import * as utilities from '../data/utilities';
+import * as processing from '../data/processing';
+
+import styles from './Animas.css';
+
+const Animas = (props) => {
+  const { bgUnits, pumpSettings } = props;
+
+  const renderBasalsData = () => {
+    const columns = [
+      { key: 'start', label: 'Start time', className: '' },
+      { key: 'rate', label: 'Value (U/hr)', className: '' },
+    ];
+    const schedules = utilities.getScheduleNames(pumpSettings.basalSchedules);
+
+    const tables = schedules.map((schedule) => {
```

Comment:
I would have preferred picking a standard here between native `.map` and `_.map`, but once again we've gone different directions. How important do we think consistency is here? It trips me up sometimes because of the different arguments (callback alone vs. callback second).